config/server.yaml

```diff
@@ -25,6 +25,12 @@ server:
     # Whether PCBIDs must be added to the network before games will work.
     enforce_pcbid: False
 
+# Webhook URLs
```

Comment:
This should probably be elaborated, like what this section enables, and that it can be left out entirely.